cfgov/apache/conf.d/redirects.conf

```diff
@@ -1019,12 +1019,10 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/protecting/?
 
 
 # Common misspellings
-
 RedirectMatch (?i)(.*)morgage(.*) $1mortgage$2
 
-
-#######################
-# Everything below this point is diverging from the production configs at
-# https://GHE/CFPB/conf.d/blob/master/redirects.conf
-
 Redirect /assets https://s3.amazonaws.com/files.consumerfinance.gov/a/assets
+
+<If "-T osenv('APACHE_COMMON_LOGIN_ALIAS')">
```

Comment:
Can kill section due to it aging out, see conversation on a-c-f

Reply:
Will do.